cost/aws/unused_ip_addresses/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 5.2
+
+- Added `resourceName` incident field
```

Comment:
Like the other CHANGELOG, this should refer to the incident fields by the name the user would see.